xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/widgets/notification.css

```diff
@@ -2,24 +2,27 @@
 .xnotification-container {
   position : fixed;
   bottom: 0;
-  left: 0%;
-  width: 100%;
+  left: 0;
+  right: 0;
+  margin: auto;
   text-align: center;
   z-index: 1200;
   display: block !important;
+  max-width: 40%;
```

Comment:
Is this OK on small screens? We might want to use a different max width in this case.

Reply:
Thank you for the review. I made this rule responsive indeed, turning it to 80% for screens narrower than 992px (@screen-md-min).